test/extended/clusterup.sh

```diff
@@ -158,6 +158,21 @@ function os::test::extended::clusterup::hostdirs () {
     BASE_DIR="${base_dir}" os::test::extended::clusterup::internal::hostdirs
 }
 
+# Tests bringing up the service catalog and provisioning a template
+function os::test::extended::clusterup::service_catalog() {
```

Comment:
talked to @bparees about this during scrum ... obtained agreement to leave these `clusterup.sh` changes in